friend-face-react/src/pages/FriendFace.js

```diff
@@ -38,7 +38,7 @@ function FriendFacePage() {
           (a, b) => Number(new Date(a.date)) - Number(new Date(b.date))
         );
       console.log(sorted);
-      setLoadedPosts(sorted);
+      //setLoadedPosts(sorted);
```

Comment:
It's generally better to delete code than to comment it out. We can always retrieve it, as long as it's in a commit somewhere.